.readme-partials.yml

```diff
@@ -71,3 +71,58 @@ body: |-
 
   If you already have a "raw" Http `request` object you can assign it to `entry.metadata.httpRequest` directly. More information about
   how the `request` is interpreted as raw can be found in the [code](https://github.com/googleapis/nodejs-logging/blob/15849160116a814ab71113138cb211c2e0c2d4b4/src/entry.ts#L224-L238).
+
+  ## Error handling with logs written or deleted asynchronously
+
+  The `Log` class provide users the ability to write and delete logs asynchronously. However, there are cases when log entries
+  cannot be written or deleted and error is thrown - if error is not handled properly, it could crash the application.
+  One possible way to catch all errors is to `await` the log write/delete calls and wrap it with `try/catch` like in example below:
+
+  ```js
+      // Write log entry and and catch any errors
+      try {
+        await log.write(entry);
+      } catch (err) {
+        console.log('Error is: ' + err);
+      }
+  ```
+
+  However, awaiting for every `log.write` or `log.delete` calls may introduce delays which could be avoided by 
+  simply adding a callback like in example below - this way the log entry can be queued for processing and code
+  execution will continue without further delays, so callback will be called once operation is complete:
```

Comment:
in the example below. This way the log entry can be queued for processing and code
  execution will continue without further delays.  The callback will be called once the operation is complete:

Reply:
Thanks!